mlflow/java/src/test/java/org/mlflow/SerializationUtilsTest.java

```diff
@@ -23,7 +23,7 @@ public void testSupportedJavaObjectsCanBeJsonSerializedAndDeserializedInTheSameF
     try {
       String listJson = SerializationUtils.toJson(inputList);
       List<Integer> loadedList = SerializationUtils.fromJson(listJson, List.class);
-      Assert.assertEquals(loadedList, inputList);
+      Assert.assertEquals(inputList, loadedList);
```

Comment:
JUnit's `assertEquals()` function parameters are of the form `assertEquals(<expected value>, <actual value>)`; these parameters were out-of-order